pkgs/top-level/python-packages.nix

```
@@ -3097,6 +3097,10 @@ in {
 
   oyaml = callPackage ../development/python-modules/oyaml { };
 
+  pam = callPackage ../development/python-modules/pam {
+    pam = pkgs.pam;
```

Could also be written as:

```suggestion
    inherit (pkgs) pam;
```